packages/karma-runner/test/unit/karma-plugins/test-hooks-middleware.spec.ts

```diff
@@ -38,6 +38,7 @@ describe(TestHooksMiddleware.name, () => {
         expect(sut.currentTestHooks)
           .contains('window.__strykerShouldReportCoverage__ = true')
           .contains('window.__stryker__.currentTestId = spec.id')
+          .contains('window.__stryker__ = window.__stryker__ || {};')
```

Comment:
😍

Reply:
added the same for respective mocha unit test